src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -153,6 +153,13 @@ class CrossDeviceLinkUI extends Component {
   sendSms = () => {
     if (this.props.sms.valid) {
       this.setState({sending: true})
+
+      // add a quick note that this will send a production SMS, so non-production
+      // environment users will need to amend any URLs that they receive
+      if (process.env.NODE_ENV !== 'production') {
+        alert(`An SMS will be sent, but the link in it will be to production, not to ${window.location.origin}`)
```

Comment:
hmm, in reality nothing will be sent if it's in staging. so the text is not really accurate.

Reply:
I've added a condition for staging, with an alert informing devs that an SMS will not be sent and link must be copy-pasted.